poi.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+const aliases = {
+  examples: path.join(__dirname, './examples')
+};
+
+/** @type {import('poi').Config} */
+module.exports = {
+  plugins: [
+    '@poi/eslint',
+    '@poi/bundle-report'
```

Comment:
We don't need a bundle report for the demo app.

Reply:
It was there before, but I agree.